inventory/sample/group_vars/k8s-cluster/k8s-cluster.yml

```diff
@@ -114,6 +114,8 @@ ndots: 2
 dns_mode: coredns
 # Set manual server if using a custom cluster DNS server
 #manual_dns_server: 10.x.x.x
+# Enable nodelocal dns cache
+enable_nodelocaldns: False
```

Comment:
Please add the variable to `tests/files/gce_centos7-flannel-addons.yml` with `true` as defined, so we test it in our CI job to make sure it works.

Reply:
Should also be defined as false in `kubernetes-defaults/defaults/main.yml`